dynaml/parser.go

```diff
@@ -4,8 +4,45 @@ import (
 	"container/list"
 	"strconv"
 	"strings"
+	
+	"github.com/cloudfoundry-incubator/spiff/yaml"
+	"github.com/cloudfoundry-incubator/spiff/debug"
 )
 
+/////////////////////////////////////////////////////////////////
+// internal helper nodes
+// used during expression parsing, they will never be contained in finally
+// parsed expression trees
+/////////////////////////////////////////////////////////////////
+
+type helperNode struct { }
+
+func (e helperNode) Evaluate(binding Binding) (yaml.Node, EvaluationInfo, bool) {
+	panic("not intended to be evaluated")
+}
+
+/*
+ * internal helper expression node to gather expression lists
+ * used for list constants and call argument lists during expression parsing
+ */
+type expressionListHelper struct {
+	helperNode
+	list []Expression
+}
+
+/*
+ * internal helper expression node to gather expression lists
+ * used for list constants and call argument lists during expression parsing
+ */
```

Comment:
This comment appears to be copied and pasted from above and does not actually describe the purpose or function of the `nameHelper` struct.